### DISCUSSION TITLE
Pico tech Pico 2000 sires’ model 2204 connecting into Visual studio Visual Basic.

### DISCUSSION THREAD

**Discussion by jmyglenn** [community/community#49340] [Other Feature Feedback, Questions, & Ideas]
Looking for someone familiar with Pico Tech Dataloggers and importing them into Visual Studio Visual Basic. If can help let me Know

**Comment by queenofcorgis** [comment]
Hi @jmyglenn,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this product, like https://www.picotech.com/products/data-logger. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by jmyglenn** [reply]
I’m looking for a software company to help me connect a peripheral into Visual Studio Visual Basic the Pico Scope 2000 series Model 2204A.
Email[redacted due to PII]

**Reply by queenofcorgis** [reply]
@jmyglenn we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍

**Comment by jmyglenn** [comment]
Not sure what to say, this. I’m trying to hire someone, to connect my Pico Scope Data Logger into Visual Studio Visual Basic. 
Pico Tech 2000 sires’ model 2204A. Pico Scope. . 
They would have to write a computer Modula to connect it. Can you help. Back in the year 2000 they use to this in a simple program now they don’t any more now their SDK only has computer Drivers In its SDK.
